blazor/3D-chart/appearance.md

```diff
@@ -0,0 +1,559 @@
+---
+layout: post
+title: Appearance in Blazor 3D Chart Component | Syncfusion
+description: Checkout and learn here all about Appearance Customization in Syncfusion Blazor 3D Chart component and much more.
+platform: Blazor
+control: 3D Chart
+documentation: ug
+---
+
+# Appearance in Blazor 3D Chart Component
+
+## Custom color palette
+
+The default color of series or points can be customized by providing a custom color palette of your choice by using the [Palettes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SfChart.html#Syncfusion_Blazor_Charts_SfChart_Palettes) property.
```

Comment:
@IlamathiSF3875 
Do not use chart APIs, you can highlight like this `Palettes` instead of [Palettes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SfChart.html#Syncfusion_Blazor_Charts_SfChart_Palettes) 
Check and change this for all API references.